src/main/shop/model/Fruit.java

```diff
@@ -0,0 +1,39 @@
+package model;
+
+import java.util.Objects;
+
+public class Fruit {
+    private String name;
+
+    public Fruit(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Fruit fruit = (Fruit) o;
+        return Objects.equals(name, fruit.name);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(name);
+    }
+
+    @Override
+    public String toString() {
+        return "Fruit{" +
+                "name='" + name + '\'' +
+                '}';
+    }
```

Comment:
let's start lines with +
```suggestion
        return "Fruit{" 
                + "name='" + name + '\'' 
                + '}';
    }
```